cmake-docker-environment/labs/subdirectory-example/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.9.1)
+project(songTest)
+
+add_subdirectory(firstDay)
+add_subdirectory(secondDay)
+
+add_library(thirdDay thirdDay/thirdDay.cpp)
```

Comment:
Out of context but I am curisous why this one was an empty CMakeLists.txt https://github.com/conan-io/cmake-training/blob/debbd1235845cf12790838f15e92004e327bdedc/cmake-docker-environment/labs/subdirectory-example/thirdDay/CMakeLists.txt 🤔

Reply:
The lesson asks students to reverse engineer the empty file from the existing files.